elements/rh-cta/rh-cta.css

```diff
@@ -180,11 +180,12 @@ svg {
   font-size: var(--rh-font-size-body-text-md, 1rem);
   border-radius: var(--rh-border-radius-default, 3px);
   border-width: var(--rh-border-width-sm, 1px);
-  padding: var(--rh-space-lg, 16px) 0 !important;
+  padding-inline: 0;
 }
 
 :host([variant]) #container ::slotted(*) {
   padding: var(--rh-space-lg, 16px) var(--rh-space-xl, 24px) !important;
+  display: inline-flex !important;
```

Comment:
@bennypowers added.

Reply:
PTAL, I think we're almost there. This might need tweaks to spacing values, but it looks solid.

Any small tweaks just commit here then yeet to prod :grin:

Reply:
![Screenshot 2023-08-10 at 1 37 21 PM](https://github.com/RedHat-UX/red-hat-design-system/assets/5090492/6a89b335-ac9b-4333-b05b-62c9130a17ab)

- In the second item in the screenshot, there is still some wrapping issue.
- In the seventh item in the screenshot, everything but the icon are aligned left.

Reply:
@bennypowers So there's no set guideline of how narrow this can go because it's relative to the word that precedes it.
![Screenshot 2023-08-23 at 9 37 46 AM](https://github.com/RedHat-UX/red-hat-design-system/assets/5090492/1fca6a87-fc68-432a-974d-b891196d0365)

Reply:
do we need to break words? can we just say "don't make the cta shorter than it's longest word plus margin"?

Reply:
ok so now we break words. good?

Reply:
Correction, even with those guidelines it could still break.
![Screenshot 2023-08-23 at 10 12 04 AM](https://github.com/RedHat-UX/red-hat-de
![Screenshot 2023-08-23 at 10 11 30 AM](https://github.com/RedHat-UX/red-hat-design-system/assets/5090492/65a1303b-4ddb-4fdf-9207-1b5401c201ab)
sign-system/assets/5090492/3608aa2c-82d5-445f-a099-5520f27789da)

Reply:
i'm still not sure what the desired behaviour is, or if CSS is currently capable of it, whatever it is :/

Reply:
resolved by [3d87c7f](https://github.com/RedHat-UX/red-hat-design-system/pull/1193/commits/3d87c7f5420267831460965ec9183be8ec75a165)